query-node/substrate-query-framework/cli/src/templates/index-builder-entry.mst

```diff
@@ -122,7 +122,9 @@ async function main() {
       "WS_PROVIDER_ENDPOINT_URI environment variable is not set! Make sure you set it in your .env file or system wide."
     );
   }
-
+  
+  // Required for warthog models.
+  dotenv.config({ path: '../graphql-server/.env' });
```

Comment:
This makes the indexer dependent on the graph server, isn't it? What env properties do we need here apart from the DB?

Reply:
Yes. Just `WARTHOG_APP_HOST` and `WARTHOG_APP_PORT` actually, DB env is the same with warthog. Should I add them to indexer's `.env` or?

Reply:
Yeah, I think we already have `templates/dotenv.mst` for that don't we?

Reply:
Probably a more serious problem here is that we somehow need to import the model files from graphql-server. I remember we already had this issue, how do we solve it?

Reply:
Yeah, we have discussed importing models but we don't have a solution yet.

Reply:
Ok, it's out of scope for this PR, but here's an issue for the future reference: https://github.com/Joystream/joystream/issues/723

Reply:
👍